clientv3/client.go

```diff
@@ -183,9 +218,21 @@ func newClient(cfg *Config) (*Client, error) {
 		c := credentials.NewTLS(cfg.TLS)
 		creds = &c
 	}
+
 	// use a temporary skeleton client to bootstrap first connection
 	ctx, cancel := context.WithCancel(context.TODO())
-	conn, err := cfg.RetryDialer(&Client{cfg: *cfg, creds: creds, ctx: ctx})
+	bootstrapClient := &Client{
```

Comment:
OK, I just introduced the bootstrap for initializing new User and Password fields. But it wouldn't be needed.